app/helpers/application_helper.rb

```diff
@@ -4,41 +4,46 @@ def asset_manager_initial_state
     portfolio = current_asset_manager.portfolio
 
     {
-      buildings: ActiveModel::Serializer::CollectionSerializer.new(
-        portfolio.buildings, each_serializer: BuildingSerializer,
-        scope: {user_id: current_asset_manager.id,
-                user_type: 'AssetManager'}
-      ),
-      userType: 'AssetManager'
+        user: current_asset_manager,
+        buildings: ActiveModel::Serializer::CollectionSerializer.new(
+            portfolio.buildings, each_serializer: BuildingSerializer,
+            scope: {user_id: current_asset_manager.id,
+                    user_type: 'AssetManager'}
```

Comment:
this is a minor thing, but instead of hardcoding `AssetManager` we could refer to the object class name: `current_asset_manager.class.name` will get you `'AssetManager'`, similar changes to below user types.